.eslintrc.json

```diff
@@ -0,0 +1,208 @@
+{
+  "parserOptions": {
+    "ecmaVersion": 2020,
+    "ecmaFeatures": {
+      "jsx": true
+    },
+    "sourceType": "module"
```

Comment:
this is only correct if you're using import/export; i'm pretty sure you want "script" here for arborist since you're not using native ESM.